bin/ci2gubernator.sh

```diff
@@ -70,7 +70,13 @@ fi
 
 if [ "${CIRCLE_BRANCH}" == "master" ] || [ "${CIRCLE_BRANCH}" == "release-1.2" ]; then
```

Comment:
This condition should only works for Postsubmit. We don't want to polute the presubmit results with some other presubmit tests we don't care.

Reply:
Done, added check for "${CI_PULL_REQUEST}" == "https://github.com/istio"*